src/gymnasium_interface/task_executor.py

```diff
@@ -0,0 +1,111 @@
+from pycram.worlds.bullet_world import BulletWorld
+from pycram.world_concepts.world_object import Object
+from pycram.datastructures.enums import ObjectType, WorldMode, Grasp
+from pycram.datastructures.pose import Pose
+from pycram.designators.action_designator import NavigateAction, PickUpAction, PlaceAction, OpenAction, CloseAction
+from pycram.designators.object_designator import BelieveObject
+from pycram.process_module import simulated_robot
+
+class PyCRAMTaskExecutor:
+    def __init__(self):
+        """Initializes task executor for PyCRAM actions."""
+        self.world = BulletWorld(WorldMode.GUI)
```

Comment:
Creating a new BulletWorld here will break everything. Use the existing world from "datastructures.World.current_world"